pkg/rewards/10_goldAvsODRewardAmounts.go

```diff
@@ -0,0 +1,78 @@
+package rewards
+
+import (
+	"github.com/Layr-Labs/sidecar/pkg/rewardsUtils"
+	"go.uber.org/zap"
+)
+
+const _10_goldAvsODRewardAmountsQuery = `
```

Comment:
In terms of ordering, this needs to come before gold staging and the gold final tables.

That means that we should rename the files (and tables) accordingly to properly reflect the order in which these queries need to happen.

Reply:
Thoughts on just calling them `goldStaging` and `goldFinal` (i.e without the numbers)? We have more reward types coming out for slashing and we'll have to refactor again in that case.

Reply:
Done in [3d27474](https://github.com/Layr-Labs/sidecar/pull/131/commits/3d27474364a3d38c25d9cdf5ddc626fe872ac984)